src/studio-header/HeaderBody.jsx

```diff
@@ -37,6 +37,7 @@ const HeaderBody = ({
   mainMenuDropdowns,
   outlineLink,
   searchButtonAction,
+  full,
```

Comment:
Same comment as https://github.com/openedx/frontend-app-course-authoring/pull/1258/files#diff-1c1e7c5912d6ff798a72899cde884c54211549e22fec1f469c8a5a4a854819e1R20 -- can this be named `fullWidth` or `fullSize` instead?

Reply:
Done: f6673e9efce2ec124d27fe7567ce63e4a845aeaf